Components/Buttons/Buttons.css

```diff
@@ -0,0 +1,33 @@
+:root{
+    --bg-primary:#0d6efd;
+    --bg-secondary:#6c757d;
+    --bg-success:#198754;
+    --bg-danger:#dc3545;
+    --bg-warning:#ffc107;
+    --bg-info:#0dcaf0;
+}
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.container{
+    max-width: 600px;
+    border: 2px solid black;
```

Comment:
Use same units , e.g border-radius is in rem but border is in px . Consistency is needed.